### ISSUE TITLE
bash hangs up after run "git log" and "CTRL+C"

### ISSUE THREAD

**Issue by ztou** [git-for-windows/git#1058]
$ git --version --build-options
```
git version 2.11.0.windows.3
built from commit: e11df2efb3072fe73153442589129d2eb8d9ea02
sizeof-long: 4
machine: x86_64
```

**reproducible steps:**

1. start bash with command:
```cmd /c "C:\Dropbox\opt\Git\bin\bash" --login -i```

2. make sure the pager is set to less 
```git config --global core.pager less```

3. go to a git repo with enough commit history and run command
```git log (make sure it only show part of you history, usually, we can type "q" to quit)```

4. press CTRL+C

**Result**: bash hangs up

I think this might be the same issue as https://github.com/git-for-windows/git/issues/227

**Comment by fourpastmidnight** [comment]
I've been everything the same with 2.11.1, but not 2.11.01. I'll fill in some details in a few hours.

**Comment by fourpastmidnight** [comment]
Here are those promised details.

So, I went to my normal installation of Git for Windows on my machine. Apparently, I had run this command around Feb 9, 9pm EST:

```
$ git log --oneline -n20
```

It still has not produced _any_ output! Hitting <kbd>CTRL+C</kbd> does terminate the running `git.exe` process. The version of Git for Windows is 2.11.1.windows.1, 64-bit on Windows 10 AU:

```
$ git --version --build-options
git version 2.11.1.windows.1
built from commit: 1c1842bcba45569a84112ec64f72b08eb2d57c68
sizeof-long: 4
machine: x86_64

$ cmd.exe /c ver

Microsoft Windows [Version 10.0.14393]

$ cat /etc/install-options.txt
Path Option: Cmd
SSH Option: OpenSSH
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Performance Tweaks FSCache: Enabled
Use Credential Manager: Enabled
Enable Symlinks: Enabled
Enable Builtin Difftool: Enabled
```

Running the Git for Windows SDK as part of the MSI installer development work, the version of Git for Windows in use is `2.10.2`. (I guess I'm a bit further behind than I anticipated).

**Comment by capt-morgan** [comment]
Thank you Ztou. I just want to highlight your workaround using 'q' to quit.

**Comment by fourpastmidnight** [comment]
So, as I was looking into Git slowness tonight, per #1071, I remembered that I had upgraded my TrendMicro Maximum Protection anti-virus and firewall booster program. I never had an issue before with it, but decided to take a look. I decided to add the path to my Git for Windows installation as an exception (i.e. no scans or other monitoring will be done on anything in the folder or below it). And lo and behold, `git log` is working again.
